<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="InicialPoint">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicial Point</string>
    </property>
   </widget>
   <widget class="QLabel" name="FinalPoint">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Point</string>
    </property>
   </widget>
   <widget class="QLabel" name="IP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="costo_labelt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Costo</string>
    </property>
   </widget>
   <widget class="QLabel" name="costo_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <widget class="QMenu" name="menuType_of_terrain">
     <property name="title">
      <string>Type of terrain</string>
     </property>
     <addaction name="actionSet"/>
    </widget>
    <widget class="QMenu" name="menuPoints">
     <property name="title">
      <string>Points</string>
     </property>
     <addaction name="actionInicial"/>
     <addaction name="actionFinal"/>
    </widget>
    <addaction name="menuType_of_terrain"/>
    <addaction name="menuPoints"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuSolve">
    <property name="title">
     <string>Solve</string>
    </property>
    <widget class="QMenu" name="menuBredth_First_Search">
     <property name="title">
      <string>Bredth First Search</string>
     </property>
     <addaction name="actionSolve"/>
     <addaction name="actionNodo_by_Node"/>
     <addaction name="actionStep_by_step"/>
    </widget>
    <widget class="QMenu" name="menuDepth_First_Search">
     <property name="title">
      <string>Depth First Search</string>
     </property>
     <addaction name="actionSolve_2"/>
    </widget>
    <widget class="QMenu" name="menuIterative_Depth_First_Search">
     <property name="title">
      <string>Iterative Depth First Search</string>
     </property>
     <addaction name="actionSolve_3"/>
    </widget>
    <widget class="QMenu" name="menuA">
     <property name="title">
      <string>A *</string>
     </property>
     <addaction name="actionSolve_4"/>
     <addaction name="separator"/>
     <addaction name="actionCosto"/>
    </widget>
    <addaction name="menuBredth_First_Search"/>
    <addaction name="menuDepth_First_Search"/>
    <addaction name="menuIterative_Depth_First_Search"/>
    <addaction name="separator"/>
    <addaction name="menuA"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionCover_map"/>
    <addaction name="actionSolve_5"/>
   </widget>
   <addaction name="menuCreate"/>
   <addaction name="menuMap"/>
   <addaction name="menuSolve"/>
   <addaction name="menuShow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="text">
    <string>Set</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>change</string>
   </property>
  </action>
  <action name="actionInicial">
   <property name="text">
    <string>Inicial</string>
   </property>
  </action>
  <action name="actionFinal">
   <property name="text">
    <string>Final</string>
   </property>
  </action>
  <action name="actionCover_map">
   <property name="text">
    <string>Cover  map</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionNodo_by_Node">
   <property name="text">
    <string>Nodo by Node</string>
   </property>
  </action>
  <action name="actionStep_by_step">
   <property name="text">
    <string>Step by step</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionSolve_2">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionSolve_3">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionSolve_4">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionCosto">
   <property name="text">
    <string>Costo</string>
   </property>
  </action>
  <action name="actionSolve_5">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
